data_table.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import numpy as np
+import dash
+from dash import dash_table, html
+
+df= pd.read_csv('csv/customers-1000.csv')
```

Comment:
Missing whitespace after `df`. 
```suggestion
df = pd.read_csv('csv/customers-1000.csv')
```

Also the variable `df` can have a better name like `customers_csv_data`